torch/_inductor/config.py

```diff
@@ -64,6 +64,9 @@
 # Optimize away split cat patterns (Experimental)
 split_cat_fx_passes = True
 
+# Optimize conv-batchnorm if batchnorm is in eval mode. Slightly reduces numerical stability.
```

Comment:
I would put it as "slightly reduced numerical stability" rather than "reduced accuracy". I have done many end-to-end training experiments in the paper, and the accuracy changes are within the range of random fluctuation. Sometimes it even gets better accuracy than the eager mode.

Reply:
The results above seemed mostly worse than eager.  Have you done end-to-end experiments on the models seeing the larger issues (e.g. swsl_resnext101_32x16d bf16)?

Reply:
I have end-to-end experiments for ResNext101 in detection tasks. Please check this [image](https://user-images.githubusercontent.com/23236638/266091025-db4815b0-d93e-4726-b1d5-e6651f256484.png) Haven't done any end-to-end training experiments aligned with torchbenchmark.

By "worse", you mean the output differs more with the fp64 version, right? But that doesn't directly translate into end-to-end performance loss. I mean, distance with the fp64 reference results and end-to-end training performance are not aligned with each other. It is possible that larger distance with fp64 reference results can get better end-to-end performance (say classification accuracy or detection mAP).

Reply:
Those experiments are pretty unconvincing to me.  It is basically just variants of 4 models, and those are not the same models we are seeing accuracy failures on in TorchBench.

A bigger difference from float64 means less accurate.  You are further away from the "true" answer.  While you might get lucky on some specific prediction, on average this should reduce model accuracy.

Reply:
Well, those experiments are done before I got familiar with torchbench 🤣 I can try to add more experiments on models failing in TorchBench, but it might take quite a long time. I have to integrate them with the [detection framework](https://github.com/open-mmlab/mmdetection) I use.

That said, at least we are done with this PR. I will get back with more results to you later.

Reply:
Sounds good.  To be clear I think reduced accuracy/precision is fine.  Many other ML techniques (float16/bfloat16/quantization/sparsity) also reduce precision and are still useful.

I would encourage you to test on more models in future papers though.  Our benchmark suites have 180+ models you can run and should be easy to setup.  Far too many academic papers only test on resnet/bert/etc, which results in overfit results that don't generalize.

Reply:
Sure, I learned a lot real-world experience during this PR. And the torchbench is a treasured place for future algorithms to test their performance extensively. Kudos to you and to the PyTorch team!